src/cmdc_tools/datasets/official/FL/data.py

```diff
@@ -0,0 +1,45 @@
+import pandas as pd
+import us
+
+from ...base import DatasetBaseNoDate
+from ..base import ArcGIS
+
+
+class Florida(DatasetBaseNoDate, ArcGIS):
+    ARCGIS_ID = "CY1LXxl9zlJeBuRZ"
+    self.state_fips = int(us.states.lookup("Florida").fips)
```

Comment:
This is close, thank you for making the change.

the only problem here is that you need to remove the `self.`. `self` is not defined in this context.

We are defining _class level attributes_ rather than _instance level_ attributes. `self` is the canonical way python refers to an instance.